### PR TITLE
Add PassIt and Psono as self hostable password manager.

### PR DESCRIPTION
**Pull request by ixeft**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [x] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/Kickball/awesome-selfhosted/issues) or [PRs](https://github.com/Kickball/awesome-selfhosted/pulls), incluing closed ones.
- [x] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [x] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 entries with syntax error(s).</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please check PR guidelines and check the boxes.</td>
    </tr>
  </tbody>
</table>

#### Syntax Issues

 Name | Entry
----|----------------------
PassIt | - [PassIt](https://passit.io/) - Simple password manage with sharing features by group and user, but no administration interface ([Demo](https://app.passit.io/), [Source Code](https://gitlab.com/passit)) `AGPL-3.0` `Python/Django/Angular` 
 

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -1010,6 +1010,8 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 - [Bitwarden](https://bitwarden.com/) - Password manager with webapp, browser extension, and mobile app. ([Source Code](https://github.com/bitwarden/core)) `AGPL-3.0` `C#`
 - [keeweb](https://keeweb.info/) - This webapp is a browser and desktop password manager compatible with KeePass databases. ([Source Code](https://github.com/keeweb/keeweb)) `MIT` `HTML5`
 - [Passbolt](https://www.passbolt.com/) - Password manager dedicated for managing passwords in a collaborative way on any Web server, using a MySQL database backend. ([Source Code](https://github.com/passbolt/passbolt_api)) `AGPL-3.0` `PHP`
+- [PassIt](https://passit.io/) - Simple password manage with sharing features by group and user, but no administration interface. ([Demo](https://app.passit.io/), [Source Code](https://gitlab.com/passit)) `AGPL-3.0` `Python/Django/Angular`
```
Comment (n8225) [inline README.md]: Change Python/Django/Angularjs to Python

File: README.md [hunk 2]
```diff
@@ -1010,6 +1010,8 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
...
+- [Psono](https://psono.com/) - A promising password managers fully featured for teams. ([Demo](https://www.psono.pw), [Source Code](https://gitlab.com/psono)) `APACHE-2.0` `Python/Django/Angularjs`
```
Comment (n8225) [inline README.md]: Change Python/Django/Angularjs to Python

File: README.md (hunk 1 above)
Comment (ixeft) [reply README.md]: Done !

File: README.md (hunk 2 above)
Comment (ixeft) [reply README.md]: Done !

**Comment by nodiscc** [comment]
Is this fully self-hostable (ie. the user controls all the software and can install it on his own machine, without requirring third-party services)? Can it be installed from source?

**Comment by ixeft** [comment]
For both software, using docker is the recommended way of using the software, but both can be install from source, they seems to be fully based on FOSS software, by my knowledge.

File: README.md [hunk 3]
```diff
@@ -1010,6 +1010,8 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 - [Bitwarden](https://bitwarden.com/) - Password manager with webapp, browser extension, and mobile app. ([Source Code](https://github.com/bitwarden/core)) `AGPL-3.0` `C#`
 - [keeweb](https://keeweb.info/) - This webapp is a browser and desktop password manager compatible with KeePass databases. ([Source Code](https://github.com/keeweb/keeweb)) `MIT` `HTML5`
 - [Passbolt](https://www.passbolt.com/) - Password manager dedicated for managing passwords in a collaborative way on any Web server, using a MySQL database backend. ([Source Code](https://github.com/passbolt/passbolt_api)) `AGPL-3.0` `PHP`
+- [PassIt](https://passit.io/) - Simple password manage with sharing features by group and user, but no administration interface. ([Demo](https://app.passit.io/), [Source Code](https://gitlab.com/passit)) `AGPL-3.0` `Python`
+- [Psono](https://psono.com/) - A promising password managers fully featured for teams. ([Demo](https://www.psono.pw), [Source Code](https://gitlab.com/psono)) `APACHE-2.0` `Python`
```
Comment (nodiscc) [inline README.md]: - Psono has detailed instructions on how to set it up without docker https://doc.psono.com/mydoc_install_server.html so there's no need to add a ` Docker` tag. Please just fix the license case `Apache-2.0`
- PassIt without docker is not really supported but has some guidance here https://passit.io/install/. So fine.

Comment (ixeft) [reply README.md]: Apache case change is done :)

**Comment by nodiscc** [comment]
Squashed and merged https://github.com/Kickball/awesome-selfhosted/commit/d18c629adeff453c6e6ad120bf652032cf225dc6
Thanks

**Pull request closed without merging**